astroid/nodes/__init__.py

```diff
@@ -31,6 +31,7 @@
     Assert,
     Assign,
     AssignAttr,
+    AssignedStmtsPossibleNode,
```

Comment:
Sorry for the confusion so far.

It doesn't really make sense to expose the type here as it's only used internally. We can use it from `astroid.nodes.node_classes` directly.